hacep/src/main/java/org/kie/hacep/core/GlobalStatus.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.hacep.core;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Shared global status
+ */
+public class GlobalStatus {
+
+    public static final AtomicBoolean nodeReady = new AtomicBoolean(false);
+    public static final AtomicBoolean nodeLive = new AtomicBoolean(true);
+    public static final AtomicBoolean canBecomeLeader = new AtomicBoolean(true);
```

Comment:
Very minor: AtomicBooleans are normally used for CAS operations and I in generally I believe that there won't be any concurrency at all, at least internally to a single node. I think they could be safely replaced with boolean, but again I'm not fussed at all about this.

Reply:
Done :+1: